crates/bevy_log/src/lib.rs

```diff
@@ -134,7 +134,69 @@ pub(crate) struct FlushGuard(SyncCell<tracing_chrome::FlushGuard>);
 ///         .run();
 /// }
 /// ```
+/// # Example Setup
 ///
+/// For a quick setup that enables all first-party logging, while not showing any of your dependencies' log data
+/// you can configure the plugin as below.
+///
+/// ```no_run
+/// # use bevy_app::{App, NoopPluginGroup as DefaultPlugins, PluginGroup};
+/// # use bevy_log::*;
+/// App::new()
+///     .add_plugins(DefaultPlugins.set(LogPlugin {
+///         filter: "warn,my_crate=trace".to_string(), //specific filters
+///         level: Level::TRACE,//Change this to be globally change levels
+///         ..Default::default()
+///         }))
+///     .run();
+/// ```
+/// The filter (in this case an `EnvFilter`) chooses whether to print the log, the most specific filters apply
+/// lets start with an example `filter: "warn".to_string()` will only print logs with level `warn` level or greater
+/// From here we can change to `filter: "warn,my_crate=trace".to_string()` logs will only print at warn unless it's in `mycrate`
+/// which will print at `trace` level `my_crate=trace` as that clause is more specific
+///
+///
+/// ## Log levels
+/// Events can be logged at various levels of importance.
+/// Only events at your configured log level and higher will be shown.
+/// ```no_run
+/// # use bevy_log::*;
+/// // here is how you write new logs at each "log level" (in "most important" to
+/// // "least important" order)
+/// error!("something failed");
+/// warn!("something bad happened that isn't a failure, but that's worth calling out");
+/// info!("helpful information that is worth printing by default");
+/// debug!("helpful for debugging");
+/// trace!("very noisy");
+/// ```
+/// In addition to `format!` style arguments you can print a variables debug
+/// value by using syntax like `trace(?my_value)`
+///
+/// ## Per module logging levels
+/// Modules can have different logging levels, using syntax like `crate_name::module_name=debug`
+///
+///
+/// ```no_run
+/// # use bevy_app::{App, NoopPluginGroup as DefaultPlugins, PluginGroup};
+/// # use bevy_log::*;
+/// App::new()
+///     .add_plugins(DefaultPlugins.set(LogPlugin {
+///         filter: "warn,my_crate=trace,my_crate::my_module=debug".to_string(), // Specific filters
+///         level: Level::TRACE, // Change this to be globally change levels
+///         ..Default::default()
+///     }))
+///     .run();
+/// ```
+/// The idea is that instead of deleting logs when they are no longer immediately applicable,
+/// you just disable them, if you need to log in future you can enable the logs instead of rewriting them.
+///
+/// ## Further reading
+/// The Tracing crate has lots more functionality than these examples can show.
```

Comment:
```suggestion
///
/// The `tracing` crate has much more functionality than these examples can show.
```
Empty line after headings + code formatting for crate names is the standard style :)